addon/components/pix-multi-select.js

```diff
@@ -17,11 +24,17 @@ export default class PixMultiSelect extends Component {
       option.checked = this._selected.includes(option.value);
     });
 
+    if(this.searchData) {
+     return defaultSelected.filter(list => 
+      this.removeCapitalizeAndDiacritics(list.label).includes(this.searchData)
```

Comment:
`removeCapitalizeAndDiacritics` n'enlève pas les capitales et les diacritics à tous les coups. C'est un peu perturbant. Peut-être que ça c'est plus clair :
```suggestion
      this.strictSearch ? list.label.includes(this.searchData) : this. removeCapitalizeAndDiacritics(list.label).includes(this.searchData)
```
Ou alors, il faut renommer `removeCapitalizeAndDiacritics` ?